config/environments/test.rb

```diff
@@ -28,7 +28,10 @@
   config.assets.digest = ENV.key?('RAILS_DISABLE_ASSET_DIGEST') ? false : true
 
   config.middleware.use RackSessionAccess::Middleware
-  config.lograge.enabled = true
+
+  # Disable lograge when computing coverage and not in CircleCI, where lograge is required.
+  # This enables scanning for view test coverage with `rake test:scan_log_for_render`
+  config.lograge.enabled = !(ENV['COVERAGE'] && !ENV['CI'])
```

Comment:
@zachmargolis I'm not sure why Circle fails w/o Lograge enabled, but this does the trick. Seem OK?

Reply:
Sure, seems like an okay check

I'd write it this way because I think it's marginally easier to read (also it appears to be an implies relation, `ENV['COVERAGE'] → ENV['CI']`)... which is the same as `!a || b`

```ruby
!ENV['COVERAGE'] || ENV['CI']
```

Reply:
Ooh, yeah.  Why is that so hard for my brain to think about? :)